Sources/Paywalls/Events/PaywallEventsManager.swift

```diff
@@ -41,18 +39,14 @@ actor PaywallEventsManager: PaywallEventsManagerType {
         internalAPI: InternalAPI,
         userProvider: CurrentUserProvider,
         store: PaywallEventStoreType,
-        appSessionID: UUID = UUID()
+        appSessionID: UUID = SystemInfo.appSessionID
     ) {
         self.internalAPI = internalAPI
         self.userProvider = userProvider
         self.store = store
         self.appSessionID = appSessionID
     }
 
-    func resetAppSessionID() {
```

Comment:
@RevenueCat/customer-center I wanted to check with you that this change is ok since it will affect customer center events. We talked over Slack and I think it's ok to do this change but want to double-check